website/docs/patterns/disabled-patterns/partials/accessibility.md

```diff
@@ -0,0 +1,21 @@
+The `disabled` attribute is only valid for these HTML elements, referred to as form controls:
+
+- `<button>`
+- `<fieldset>`
+- `<optgroup>`
+- `<option>`
+- `<select>`
+- `<textarea>`
+- `<input>`
+
+In terms of accessibility, however, a form control with the `disabled` attribute is problematic. If a form control has the `disabled` attribute, it's not available to users with assistive technology. It's as though the element does not exist in the DOM at all.
+
+When presenting a non-editable field, consider using the `readonly` attribute instead of the `disabled` attribute. This will provide the correct information to all users, but they won't be able to interact with it.
```

Comment:
Not all the controls mentioned above [support](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#overview) the `readonly` attribute. Should we clarify this?

Reply:
+1

Reply:
Maybe we should say...

```suggestion
When presenting a non-editable field, consider using the `readonly` attribute where applicable instead of the `disabled` attribute. This will provide the correct information to all users, but they won't be able to interact with it.
```